shared/wallets/receive-modal/index.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import * as React from 'react'
+import {Box2, Button, CopyText, Icon, InfoNote, MaybePopup, Text} from '../../common-adapters'
+import {globalMargins, isMobile, platformStyles, styleSheetCreate} from '../../styles'
+
+type Props = {
+  onClose: () => void,
+}
+
+const ReceiveModal = (props: Props) => (
+  <MaybePopup onClose={props.onClose}>
+    <Box2 direction="vertical" style={containerStyle}>
+      <Icon
+        type={isMobile ? 'icon-wallet-receive-64' : 'icon-wallet-receive-48'}
+        style={{marginBottom: globalMargins.small}}
+      />
+      <Text type="BodySmallSemibold">cecileb’s wallet</Text>
+      <Text type="Header" style={{marginBottom: globalMargins.medium}}>
+        Receive
+      </Text>
+      <Text
+        type="Body"
+        style={{
+          marginBottom: globalMargins.medium,
+          textAlign: 'center',
+        }}
+      >
+        To receive Lumens or assets from non-Keybase users, pass your Stellar addresses around:
+      </Text>
+      <Box2
+        direction="vertical"
+        style={{
+          marginBottom: globalMargins.tiny,
+          width: '100%',
+        }}
+      >
+        <CopyText text="G23T5671ASCZZX09235678ASQ511U12O91AQ" />
```

Comment:
I think the idea is to make this component close to the real thing, i.e. have a prop for the Stellar address, and also the e-mail address below, and the keybase name above

Reply:
I wasn't sure how close to the real thing these are supposed to be when we're implementing the storybook. I'll add those props. Thanks!